src/Identifier/IdentifierCollection.php

```diff
@@ -18,6 +18,10 @@
 use Cake\Core\App;
 use RuntimeException;
 
+/**
+ * Class IdentifierCollection
+ * @method \Authentication\Identifier\IdentifierInterface get($name)
```

Comment:
Can we annotate the type of $name? string? I think so.

Reply:
Agree. Please change.

> Doesnt the IdentifierInterface have the get() method? Then this shouldnt be needed IMO.
You are right, this method exists in AbstractCollection extends ObjectRegistry, but it exists with general type (object|null), so autocomplete for IdentifierCollection methods does not work

Reply:
The |null part though is then still true?
I feel like we should have an internal method as wrapper that also throws exception in null case then.

Reply:
Yes, agree. Not sure if its possible to change it in submitted PR.


> The |null part though is then still true?

Correct, I forgot about that.